spec/features/data_access/service_options_with_boundingbox1_spec.rb

```diff
@@ -0,0 +1,59 @@
+# EDSC-1233
+# Some forms have boundingbox1 in their echoforms instead of boundingbox
+# This was causing that field to be null in the service requests
+
+require 'spec_helper'
+require 'rake'
+
+describe 'Service options order with boundingbox1 field', reset: false do
+  collection_id = 'C1000000445-NSIDC_ECS'
+
+  context 'when placing a service order with a collection with a boundingbox1 field' do
+    before :all do
+      Delayed::Worker.delay_jobs = true
+
+      load_page :search, focus: collection_id
+      login
+      wait_for_xhr
+
+      first_granule_list_item.click_link "Retrieve single granule data"
+      choose 'GLAH06.34 ESI Service'
+
+      check 'Enter bounding box'
+      wait_for_xhr
+
+      expect(page).to have_checked_field('Enter bounding box')
+      fill_in 'Email Address', with: 'test@email.com'
+      fill_in 'North', with: '-63'
+      fill_in 'West', with: '-139'
+      fill_in 'East', with: '-100'
+      fill_in 'South', with: '-76'
+      wait_for_xhr
+
+      within '.access-item-actions' do
+        sleep 2
```

Comment:
I'm not 100% sure we do, I copied the majority of the test from spec/features/data_access/service_options_with_bad_bounding_box_spec.rb